### PR TITLE
Adds track choice suggestions

### PR DESCRIPTION
**Pull request by I3uckwheat**
Adds some advice on how to choose a track effectively. 

~~I know this doesn't exactly round out the points you laid out in the issue, but I believe this is a very important topic for the student to understand and still covers the points you've made in a roundabout way. If this is not acceptable, please let me know and I will re-write.~~

As per @KevinMulhern's executive suggestion, I have added some information about each path's technology.

#21806

### CODE REVIEW

File: web_development_101/tying_it_all_together/conclusion.md [hunk 1]
```diff
@@ -4,8 +4,14 @@ You should be fairly comfortable typing commands into your computer's command li
 
 With what you know already, you could start putting together some basic websites or strike out to increase your knowledge on your own.  However, we're here to help you get a lot further than that, to the point where you can confidently build a full-featured and scalable web application, whether on your own or working as a developer in a top-tier tech company.  
 
-You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do!
+You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do, and now is the time to decide which path you want to take. Either Ruby with Ruby on Rails or JavaScript with Node/Express. Both paths will guide you in building skills, building up a portfolio of projects and become a great developer. There is no right answer here on what is the next best path. Learning a differnet language can help you become a more well rounded developer, but sticking with one the whole way though could make you more of an expert in the lanaguage itself. Both paths will help you become a great developer. The important thing is to not switch halfway though, this will only set you back.
```
Comment (xandora) [inline web_development_101/tying_it_all_together/conclusion.md:7]: "become a great developer." -> "**becoming** a great developer."

Comment (xandora) [inline web_development_101/tying_it_all_together/conclusion.md:7]: "Learning a differnet language" -> "Learning a **different** language"

Comment (xandora) [inline web_development_101/tying_it_all_together/conclusion.md:7]: "whole way though could make" -> "whole way **through** could make"

File: web_development_101/tying_it_all_together/conclusion.md [hunk 2]
```diff
@@ -4,8 +4,14 @@ You should be fairly comfortable typing commands into your computer's command li
...
 
-Buckle up, because this is only the beginning. Now it's time to delve deeper into the areas covered by Web Dev 101. There are some fantastic projects in the courses that follow which will challenge you to the limit, but you'll come out the other side ready to go and make your mark in the programming world.
+Learning how to program is **much** more than just learning a language. It's about learning how to research and solve difficult problems, build an engineering mindset, work on complex problems and codebases, communicate effectively with other developers, and more. Luckly all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, it will be easy to move into a new language with far less difficulty than learning your first one.
```
Comment (xandora) [inline web_development_101/tying_it_all_together/conclusion.md:9]: "build an engineering mindset, work on complex problems and codebases, communicate effectively with other developers, and more."

-> "building"
-> "working"
-> "communicating"
-> "and **much** more."

File: web_development_101/tying_it_all_together/conclusion.md [hunk 3]
```diff
@@ -4,8 +4,14 @@ You should be fairly comfortable typing commands into your computer's command li
...
 
-Buckle up, because this is only the beginning. Now it's time to delve deeper into the areas covered by Web Dev 101. There are some fantastic projects in the courses that follow which will challenge you to the limit, but you'll come out the other side ready to go and make your mark in the programming world.
+Learning how to program is **much** more than just learning a language. It's about learning how to research and solve difficult problems, build an engineering mindset, work on complex problems and codebases, communicate effectively with other developers, and more. Luckly all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, it will be easy to move into a new language with far less difficulty than learning your first one. 
+
+Arguments about learning the languages used in your area may sound convincing, but because a _good_ developer can transfer skills easily, what you learn isn't going to change outcomes of getting a job much. Though, if you cannot shake that fear, do whatever you need to just stay on the train and finish the course.
```
Comment (xandora) [inline web_development_101/tying_it_all_together/conclusion.md:11]: "change outcomes" -> "change the outcome"

Comment (xandora) [inline web_development_101/tying_it_all_together/conclusion.md:11]: "do whatever you need to just stay on the train and finish the course."
->
"do whatever you need to do just to stay on the train and finish the course."

**Review by xandora — APPROVED** [review]
looks good to me, just a few little spelling and grammatical/flow issues.

**Comment by GraceSchwar** [comment]
" it will be easy to move into a new language with far less difficulty than learning your first one. " 

this sounds kinda funny. Maybe: It will be a lot easier/less difficult learning new languages after learning your first.

File: web_development_101/tying_it_all_together/conclusion.md (hunk 2 above)
Comment (I3uckwheat) [reply web_development_101/tying_it_all_together/conclusion.md]: I was going a different direction on this at first, thanks for fixing :)

**Comment by I3uckwheat** [comment]
> " it will be easy to move into a new language with far less difficulty than learning your first one. "
> 
> this sounds kinda funny. Maybe: It will be a lot easier/less difficult learning new languages after learning your first.
I have edited :)

File: web_development_101/tying_it_all_together/conclusion.md [hunk 4]
```diff
@@ -4,8 +4,40 @@ You should be fairly comfortable typing commands into your computer's command li
 
 With what you know already, you could start putting together some basic websites or strike out to increase your knowledge on your own.  However, we're here to help you get a lot further than that, to the point where you can confidently build a full-featured and scalable web application, whether on your own or working as a developer in a top-tier tech company.  
 
-You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do!
+You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do, and now is the time to decide which path you want to take. Either Ruby with Ruby on Rails or JavaScript with Node/Express. Both paths will guide you in building skills, building up a portfolio of projects and becoming a great developer. There is no right answer here on what is the next best path. Learning a different language can help you become a more well rounded developer, but sticking with one the whole way through could make you more of an expert in the lanaguage itself. Both paths will help you become a great developer. The important thing is to not switch halfway though, this will only set you back.
```
Comment (mistercwood) [inline web_development_101/tying_it_all_together/conclusion.md:7]: "in the **lanaguage** itself" becomes "in the **language** itself"

Comment (I3uckwheat) [reply web_development_101/tying_it_all_together/conclusion.md]: Fixed, thanks!

**Comment by GraceSchwar** [comment]
“strike out to increase your knowledge on your own“ — not quite sure what strike out means (others may not)


“ Learning a different language can help you become a more well rounded developer, but sticking with one the whole way through could make you more of an expert in the language itself”
Maybe: learning multiple languages can make you a more well rounded developer ... rest

**Review by zachmmeyer — APPROVED** [review]

**Comment by GraceSchwar** [comment]
Link to @bycdiaz article about why it doesn’t matter which in the portion where you mention how it doesn’t matter which language you chose cuz it gives examples of how languages are similar. (People who don’t know other languages probably won’t believe you without examples (me))

**Comment by bycdiaz** [comment]
I'm wondering if we could avoid phrasing like `left up to your own devices` or `buckle up`.

I suspect most folks on this  thread are aware of the meaning behind these phrases but we do get a fair amount of learners who struggle to understand the meaning of phrases like this.

If y'all are cool with that, I'm happy to give this a pass to exchange those kinds of phrases with stuff that might be a little more straightforward.

**Review by KevinMulhern — APPROVED** [review]
LGTM thanks @I3uckwheat 💪

**Review by marvingay — APPROVED** [review]
Took it upon myself to add a couple of missing commas directly and using the more common spelling for "Luckily", which was previously "Luckly"

LGTM!

**Comment by I3uckwheat** [comment]
> “strike out to increase your knowledge on your own“ — not quite sure what strike out means (others may not get it either)
> 
> “ Learning a different language can help you become a more well rounded developer, but sticking with one the whole way through could make you more of an expert in the language itself”
> Maybe: learning multiple languages can make you a more well rounded developer ... rest

I have made edits to this




> Link to @bycdiaz article about why it doesn’t matter which language in the portion where you mention how it doesn’t matter which language you chose cuz it gives examples of how languages are similar. (People who don’t know other languages probably won’t believe you without examples (me))
> 
> https://medium.com/@bycdiaz/choosing-the-right-language-a-short-guide-on-how-not-to-ruin-your-career-2b353be1371

I have added this




> I'm wondering if we could avoid phrasing like `left up to your own devices` or `buckle up`.
> 
> I suspect most folks on this thread are aware of the meaning behind these phrases but we do get a fair amount of learners who struggle to understand the meaning of phrases like this.
> 
> If y'all are cool with that, I'm happy to give this a pass to exchange those kinds of phrases with stuff that might be a little more straightforward.

Because I was editing, I made some changes. Feel free to change some more if you feel the need.

**Review by bycdiaz — APPROVED** [review]

**Pull request merged**